### PR TITLE
Use strict typing

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: lib/DatabaseStatistics.php [hunk 1]
```diff
@@ -88,9 +91,9 @@ protected function databaseVersion() {
 	 * @copyright (c) phpBB Limited <https://www.phpbb.com>
 	 * @license GNU General Public License, version 2 (GPL-2.0)
 	 *
-	 * @return int|string
+	 * @return string
 	 */
-	protected function databaseSize() {
+	protected function databaseSize(): string {
```
Comment (blizzz) [inline lib/DatabaseStatistics.php:96]: The method might return `$database_size`  as an int  currently. Probably cast it in the return statement.

**Review by blizzz — CHANGES_REQUESTED** [review]
one method returns mixed yet

File: lib/DatabaseStatistics.php (hunk 1 above)
Comment (J0WI) [reply lib/DatabaseStatistics.php:96]: fixed

File: lib/Os.php [hunk 2]
```diff
@@ -44,45 +47,59 @@ public function __construct() {
...
+	/**
+	 * @return Memory
+	 */
 	public function getMemory(): Memory {
 		return $this->backend->getMemory();
 	}
 
+	/**
+	 * @return string
+	 */
```
Comment (blizzz) [inline lib/Os.php:80]: when we have return type hints, we don't need phpdoc just for this. not a blocker, for next time.

**Comment by blizzz** [comment]
still needs a rebase, there's a conflict with  lib/StorageStatistics.php

**Review by blizzz — APPROVED** [review]

**Pull request merged**